spec/json_matchers/match_json_schema_spec.rb

```diff
@@ -59,6 +59,147 @@
     expect(json).to match_json_schema("api/v1/schema")
   end
 
+  it "Fails properly in local refs in referenced schema" do
+    create(:schema, name: "post", json: {
+      "$schema": "https://json-schema.org/draft-04/schema#",
+      "id": "file:/post.json#",
+      "definitions": {
+        "attributes": {
+          "type": "object",
+          "required": [
+            "id",
+            "name",
+          ],
+          "properties": {
+            "id": { "type": "number" },
+            "name": { "type": "string" }
+          }
+        }
+      },
+      "type": "object",
+      "required": [
+        "id",
+        "type",
+        "attributes"
+      ],
+      "properties": {
+        "id": { "type": "number" },
```

Comment:
@seanpdoyle The type of id should be "string". The test fails if we change the type from number to string.